src/init/starship.elv

```diff
@@ -29,5 +29,13 @@ edit:prompt = {
     }
 }
 
-# Get rid of default rprompt
-edit:rprompt = { }
+edit:rprompt = {
+    # Note:
+    # Elvish does not appear to support exit status codes (--status)
+
+    if (== $cmd-start-time 0) {
+        ::STARSHIP:: prompt --right --jobs=$num-bg-jobs
+    } else {
+        ::STARSHIP:: prompt --right --jobs=$num-bg-jobs --cmd-duration=(- $cmd-end-time $cmd-start-time)
```

Comment:
**nitpick:** Instead of evaluating `cmd-duration` once in each prompt, it may be best to evaluate it, store it as a variable, and pass it to both `starship prompt` commands.